data/time_series_errors.go

```diff
@@ -0,0 +1,11 @@
+package data
+
+import "errors"
+
+var (
+	ErrorNullTimeValues = errors.New("unable to process the data to wide series because input has null time values, make sure all time values are not null")
+	ErrorSeriesUnsorted = errors.New("unable to process the data because it is not sorted in ascending order by time, please updated your query to sort the data by time and try again")
```

Comment:
```suggestion
	ErrorSeriesUnsorted = errors.New("unable to process the data because it is not sorted in ascending order by time, please updated your query to sort the data by time if supported")
```